### PR TITLE
New stuff OneStar Structures

### PR DESCRIPTION
**Pull request by TorinoFermic**
## About The Pull Request

This PR will just add new structures and closets to this master.

## Changelog
:cl:
add: New structures for new derelicts.
imageadd: Various new structures sprites.
/:cl:

## Screenshot
image
image

### CODE REVIEW

File: code/game/objects/structures/crates_lockers/closets/onestar.dm [hunk 1]
```diff
@@ -0,0 +1,241 @@
+//Closets full of loot, they should be placed in derelicts
+
+//// Loot table on all tiers
+// Tier 1
+/obj/structure/closet/onestar/tier1
+	name = "\improper OneStar forgotten closet"
+	desc = "Old OneStar closet, it may contain junk loot."
```
Comment (drexample) [inline code/game/objects/structures/crates_lockers/closets/onestar.dm]: desc = "Old OneStar closet, doesn't look like it contains anything worthwhile." or something along those lines. 
Sounds better, more IC-ly. 
Same for the closet descriptions below.

File: code/game/objects/structures/stool_bed_chair_nest/chairs.dm [hunk 2]
```diff
@@ -274,4 +274,13 @@
...
 	name = "bar chair"
 	desc = "Modern design and soft pad. Served up with the drink and great company."
-	icon_state = "bar_chair"
\ No newline at end of file
+	icon_state = "bar_chair"
+
+//onestar
+/obj/structure/bed/chair/custom/onestar
+	name = "onestar chair"
+	desc = "Advanced chair and it looks pristine after the Fall."
```
Comment (drexample) [inline code/game/objects/structures/stool_bed_chair_nest/chairs.dm]: ...how can a chair be advanced? The sprite even looks like a normal chair.

**Comment by Kurgis** [comment]
Just noticed this was marked for review. Shazbot posted an update of the One Star lockers onto assets_storage.

File: code/game/objects/structures/crates_lockers/closets/onestar.dm [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+	new /obj/random/lowkeyrandom/low_chance(src)
+	new /obj/random/pack/tech_loot/low_chance(src)
+	new /obj/random/pack/cloth/low_chance(src)
+	new /obj/random/pack/cloth/low_chance(src)
+	new /obj/random/pack/gun_loot/low_chance(src)
+
+// Tier 2
+/obj/structure/closet/onestar/tier2
+	name = "\improper OneStar forgotten closet"
+	desc = "Old OneStar closet, there's good stuff inside it."
```
Comment (drexample) [inline code/game/objects/structures/crates_lockers/closets/onestar.dm]: Again, the characters don't know what is inside, they might only assume. 
"Old OneStar closet, it looks like there might be something good inside." or something along those lines.
They are essentially lootboxes and you don't know what's inside of them, before you actually open them.
Same goes for the tier 3, as the characters have no clue what's inside, they can only see it looks like something that people would shove valuables into.

File: code/game/objects/structures/stool_bed_chair_nest/chairs.dm [hunk 4]
```diff
@@ -274,4 +274,13 @@
...
 	name = "bar chair"
 	desc = "Modern design and soft pad. Served up with the drink and great company."
-	icon_state = "bar_chair"
\ No newline at end of file
+	icon_state = "bar_chair"
+
+//onestar
+/obj/structure/bed/chair/custom/onestar
+	name = "onestar chair"
+	desc = "Plain-looking chair. Made by OneStar."
```
Comment (drexample) [inline code/game/objects/structures/stool_bed_chair_nest/chairs.dm]: The description is too plain, even for a plain chair. Make it sound like actual flavor text, like other furniture objects for example.

**Comment by Kurgis** [comment]
I believe the One Star Alliance should be instead referenced as the One Star Hegemony. Often we create objects in-game which are otherwise mundane (like a chair), but feel the need to differentiate that the chair was of One Star manufacture. One Star Alliance doesn't have a very good possessive noun, so often things get placed with One Star locker, or One Star drill. We could use the word hegemonic instead, like "hegemonic chair", instead of "/improper One Star chair".

File: code/game/objects/structures/stool_bed_chair_nest/chairs.dm (hunk 4 above)
Comment (TorinoFermic) [reply code/game/objects/structures/stool_bed_chair_nest/chairs.dm]: Updated

File: code/game/objects/structures/stool_bed_chair_nest/chairs.dm [hunk 5]
```diff
@@ -274,4 +274,13 @@
...
 	name = "bar chair"
 	desc = "Modern design and soft pad. Served up with the drink and great company."
-	icon_state = "bar_chair"
\ No newline at end of file
+	icon_state = "bar_chair"
+
+//onestar
+/obj/structure/bed/chair/custom/onestar
+	name = "onestar chair"
+	desc = "Duranium chair. The chair uses ancient metals in his construction and is manufactured inside OneStar"
```
Comment (drexample) [inline code/game/objects/structures/stool_bed_chair_nest/chairs.dm]: Poor grammar. 
Use "A duranium chair manufactured by OneStar. Doesn't look very comfortable."

File: code/game/objects/structures/crates_lockers/closets/onestar.dm [hunk 6]
```diff
@@ -0,0 +1,241 @@
+//Closets full of loot, they should be placed in derelicts
+
+//// Loot table on all tiers
+// Tier 1
+/obj/structure/closet/onestar/tier1
+	name = "\improper OneStar forgotten closet"
+	desc = "Old OneStar closet, doesn't look like it contains anything worthwhile. Maybe."
```
Comment (drexample) [inline code/game/objects/structures/crates_lockers/closets/onestar.dm]: "It's an old OneStar closet. Doesn't seem like it contains anything worthwhile. Probably."

File: code/game/objects/structures/crates_lockers/closets/onestar.dm [hunk 7]
```diff
@@ -0,0 +1,241 @@
...
+	new /obj/random/lowkeyrandom/low_chance(src)
+	new /obj/random/pack/tech_loot/low_chance(src)
+	new /obj/random/pack/cloth/low_chance(src)
+	new /obj/random/pack/cloth/low_chance(src)
+	new /obj/random/pack/gun_loot/low_chance(src)
+
+// Tier 3
+/obj/structure/closet/onestar/tier3
+	name = "\improper OneStar forgotten closet"
+	desc = "Old OneStar closet, may contain very valuable items or nothing of worthwhile."
```
Comment (drexample) [inline code/game/objects/structures/crates_lockers/closets/onestar.dm]: "It's an old OneStar closet. Might contain something very valuable, or so you hope."

File: code/game/objects/structures/crates_lockers/closets/onestar.dm [hunk 8]
```diff
@@ -0,0 +1,241 @@
...
+	new /obj/random/lowkeyrandom/low_chance(src)
+	new /obj/random/pack/tech_loot/low_chance(src)
+	new /obj/random/pack/cloth/low_chance(src)
+	new /obj/random/pack/cloth/low_chance(src)
+	new /obj/random/pack/gun_loot/low_chance(src)
+
+// Tier 2
+/obj/structure/closet/onestar/tier2
+	name = "\improper OneStar forgotten closet"
+	desc = "Old OneStar closet, it might have some good stuff inside it."
```
Comment (drexample) [inline code/game/objects/structures/crates_lockers/closets/onestar.dm]: "It's an old OneStar closet. Looks like there might be some decent stuff inside."

**Pull request merged**